releases/links_v2.0.0/LINKS-make

```diff
@@ -0,0 +1,142 @@
+	@echo "		clean           remove intermediate files"
+	@echo ""
+	@echo "	General Options:"
+	@echo ""
+	@echo "		draft           draft name [draft]. File must have .fasta or .fa extension"
+	@echo "		readsFof        file of files for reads [reads fof]. Files must have .fastq.gz or .fq.gz extension"
+	@echo "		time		logs time and memory usage to file for main steps (Set to 1 to enable logging)"	
+	@echo ""
+	@echo "	LINKS Options:"
+	@echo ""
+	@echo "	m	MPET reads (default -m 1 = yes, default = no, optional"
+	@echo "			! DO NOT SET IF NOT USING MPET. WHEN SET, LINKS WILL EXPECT A SPECIAL FORMAT UNDER -s"
+	@echo "			! Paired MPET reads in their original outward orientation <- -> must be separated by ":""
+	@echo "			>template_name"
+	@echo "			ACGACACTATGCATAAGCAGACGAGCAGCGACGCAGCACG:ATATATAGCGCACGACGCAGCACAGCAGCAGACGAC"
+	@echo "	d	distance between k-mer pairs (ie. target distances to re-scaffold on. default -d 4000, optional)"
+	@echo "			Multiple distances are separated by comma. eg. -d 500,1000,2000,3000"
+	@echo "	k	k-mer value (default -k 15, optional)"
+	@echo "	t	step of sliding window when extracting k-mer pairs from long reads"
+	@echo "			(default -t 2, optional)"
+	@echo "			Multiple steps are separated by comma. eg. -t 10,5"
+	@echo " j	threads  (default -j 3, optional)"
+	@echo "	o	offset position for extracting k-mer pairs (default -o 0, optional)"
+	@echo "	e	error (%) allowed on -d distance   e.g. -e 0.1  == distance +/- 10%"
+	@echo "		(default -e 0.1, optional)"
+	@echo "	l	minimum number of links (k-mer pairs) to compute scaffold (default -l 5, optional)"
+	@echo "	a	maximum link ratio between two best contig pairs (default -a 0.3, optional)"
+	@echo "			*higher values lead to least accurate scaffolding*"
+	@echo "	z	minimum contig length to consider for scaffolding (default -z 500, optional)"
+	@echo "	b	base name for your output files (optional)"
+	@echo "	r	Bloom filter input file for sequences supplied in -s (optional, if none provided will output to .bloom)"
+	@echo "			NOTE: BLOOM FILTER MUST BE DERIVED FROM THE SAME FILE SUPPLIED IN -f WITH SAME -k VALUE"
+	@echo "			IF YOU DO NOT SUPPLY A BLOOM FILTER, ONE WILL BE CREATED (.bloom)"
+	@echo "	p	Bloom filter false positive rate (default -p 0.001, optional; increase to prevent memory allocation errors)"
+	@echo "	x	Turn off Bloom filter functionality (1 = yes, default = no, optional)"
+	@echo "	v	Runs in verbose mode (1 = yes, default = no, optional)"
+	@echo ""
+
+clean:
+	rm -f *.pairing_issues *.tsv
+	@echo "Clean Done"
+
+version:
+	@echo "links-make v2.0.0"
+
+# Make an Empty fof File
+empty.fof:
+	touch $@
+
+$(b).tigpair_checkpoint.tsv : $(draft) $(readsFof)
+	$(gtime) $(bin)/src/LINKS_CPP -f $(draft) -s $(readsFof) $(r_conditional) $r -d $d -k $k -t $t -o $o -e $e -z $z -p $p -b $b -v $v -x $x
+
+$(b).scaffold.fa : $(b).tigpair_checkpoint.tsv empty.fof
+	$(gtime) perl $(bin)/src/LINKS.pl -f $(draft) -s empty.fof -b $b -l $l -a $a -v $v
```

Comment:
Can we be sure that all the parameters that ARCS specifies in its makefile are here too? Looks like `-z` is missing

Reply:
OK I'll check I though that param wasnt used in perl stage